_applications/equoid.adoc

```diff
@@ -0,0 +1,206 @@
+= Equoid
```

Comment:
The other Tutorials have been given more verbose titles which give more insight into what the tutorial is doing. Is this something that we should do for Equoid too? eg. "Equoid: a top K tracking system"?

Reply:
agreed, but i think we should push a little further. my preference is to leave "equoid" as the project codename and then have the title be something that gives the reader a quick impression of why or how they might use this in a _real world_ situation.

i'm not sure if putting "top-k" in the title will fit that, i think there are 2 strong uses that come out of this tutorial: the top-k clustering, and space saving of count-min sketching. not sure how, or if, we should combine these but it's more food for thought.

Reply:
A possible alternative to "top-k" might also be "trending topic detection" or "trending item detection"

Reply:
what about just "scalable leaderboards?"

Reply:
i like the direction this discussion is going

> Scalable trending item detection of streaming data ?

++

Reply:
after thinking about this a little more, i wonder if "Scalable trending item detection *on* streaming data" might read a little better?

Reply:
Good suggestion